includes/classes/sniffer.php

```diff
@@ -74,5 +74,40 @@ function field_type($table_name, $field_name, $field_type, $return_found = false
     }
     return false;
   }
+
+  /**
+   * Return true if the specified row exists in the table.
+   * Either pass a string $key_name and int $key_value, or arrays for both.
+   * If arrays are passed, each element in $key_name will be matched against
+   * its corresponding index entry in $key_value, similar to how str_replace()
+   * works.
+   *
+   * @param [string] $table_name The table to query.
+   * @param [string|array] $key_name The key, or array of keys, to check.
+   * @param [int|array] $key_value The value, or array of values, that key_name must equal.
+   * @return void
+   */
+  function row_exists(string $table_name, string|array $key_name, int|array $key_value) : bool {
```

Comment:
- Use `public function`
- Opening brace on new line
- No space between ending parenthesis and `:`
- Remove `union types`, not supported until PHP 8.0

Reply:
Ah gotcha, thanks for the patience. I've pushed a fix for most issues. (also installed a PSR-12 linter to make this easier in future, though vscode integration is a bit clunky)

On the other two, the sniffer.php file is aready using 2 spaces for indent and function names like `table_exists` not `tableExists` .. should I copy that, or use new standards in my addition only, or convert the whole file to the newer standard?